src/docbuilder/rustwide_builder.rs

```diff
@@ -469,6 +537,50 @@ impl RustwideBuilder {
         Ok(())
     }
 
+    fn get_coverage(
+        &self,
+        target: &str,
+        build: &Build,
+        metadata: &Metadata,
+        limits: &Limits,
+    ) -> Result<Option<DocCoverage>> {
+        let mut doc_coverage_json = None;
+
+        let rustdoc_flags = vec![
+            "--output-format".to_string(),
+            "json".to_string(),
+            "--show-coverage".to_string(),
+        ];
```

Comment:
Shouldn't this also include the rustdoc flags in `metadata`? Otherwise it might be missing a `--cfg` which would have different output.

Reply:
Done!